### PR TITLE
feat(env): adds hospitalrun server information

### PR DESCRIPTION
**Pull request by jackcmeyer**
**Changes proposed in this pull request:**

- adds a `.env.example` file with example configurations or the HospitalRun Server
- adds environment variable in pouchdb configuration

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_HOSPITALRUN_SERVER=http://0.0.0.0:3001
```
Comment (fox1t) [inline .env.example]: Change to `HOSPITALRUN_API`. We will use the same war in CD/CI and Kubernetes.

File: README.md [hunk 2]
```diff
@@ -27,6 +27,15 @@ Contributions are always welcome. Before contributing please read our [contribut
...
 4. Run `yarn run start` to build and watch for code changes
 
+## Connecting to HospitalRun Server
+
+__Note: The following instructions are for connecting to HospitalRun Server during development and are not intended to be for production use. For production deployments, see the deployment instructions.__
+
+1. Configure [HospitalRun Server](https://github.com/HospitalRun/hospitalrun-server)
+2. Start the HospitalRun Development Server
+3. Copy the `.env.example` file to `.env.local` or `.env`
+4. Change the `REACT_APP_HOSPITALRUN_SERVER` variable to point to the HospitalRun Development Server.
```
Comment (fox1t) [inline README.md]: Change to `HOSPITALRUN_API`

File: README.md [hunk 3]
```diff
@@ -27,6 +27,15 @@ Contributions are always welcome. Before contributing please read our [contribut
 3. Install the dependencies: `yarn`
 4. Run `yarn run start` to build and watch for code changes
 
+## Connecting to HospitalRun Server
+
+__Note: The following instructions are for connecting to HospitalRun Server during development and are not intended to be for production use. For production deployments, see the deployment instructions.__
+
+1. Configure [HospitalRun Server](https://github.com/HospitalRun/hospitalrun-server)
+2. Start the HospitalRun Development Server
+3. Copy the `.env.example` file to `.env.local` or `.env`
```
Comment (fox1t) [inline README.md]: Leave only `.env` so we will have uniformity on all projects.

File: src/config/pouchdb.ts [hunk 4]
```diff
@@ -16,15 +16,12 @@ function createDb(name: string) {
...
-  // db.sync(
-  //   `https://a27fa3db-db4d-4456-8465-da953aee0f5b-bluemix:cd6f332d39f24d2b7cfc89d82f6836d46012ef3188698319b0d5fff177cb2ddc@a27fa3db-db4d-4456-8465-da953aee0f5b-bluemix.cloudantnosqldb.appdomain.cloud/${name}`,
-  //   {
-  //     live: true,
-  //     retry: true,
-  //   },
-  // ).on('change', (info) => {
-  //   console.log(info)
-  // })
+  db.sync(`${process.env.REAC_APP_HOSPITALRUN_SERVER}/_db/${name}`, {
```
Comment (fox1t) [inline src/config/pouchdb.ts]: change to`HOSPITALRUN_API`

File: .env.example (hunk 1 above)
Comment (jackcmeyer) [reply .env.example]: Sounds good!!

**Comment by jackcmeyer** [comment]
@fox1t, I have addressed your changes. This PR is ready for re-review!

**Comment by tehKapa** [comment]
@jackcmeyer some tests fail

**Pull request merged**